gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -134,6 +134,17 @@ public static GoogleCloudStorageGrpcReadChannel open(
         BackOffFactory.DEFAULT);
   }
 
+  public static GoogleCloudStorageGrpcReadChannel open(
```

Comment:
Let's keep this package-private

Reply:
It is being used in GoogleCloudStorageImpl and hence cannot be kept package-private

Reply:
GoogleCloudStorageImpl and ReadChannel are in same package